scripts/syncPostsToGrapher.ts

```diff
@@ -1,14 +0,0 @@
-import * as wpdb from "db/wpdb"
-import * as db from "db/db"
-import { syncPostsToGrapher } from "db/model/Post"
-
-async function main() {
-    try {
-        await syncPostsToGrapher()
-    } finally {
-        await wpdb.end()
-        await db.end()
-    }
-}
-
-main()
```

Comment:
I think Matthieu already mentioned – very rarely this is useful because the postUpdate script fails. I would keep it